index.js

```diff
@@ -13,6 +15,13 @@ app.set('view engine', 'hbs');
 
 app.use(logger('dev'));
 
+app.use(session({
+  name: 'izels-session-cookie',
+  secret: 'our secret key',
+  resave: true,
+  saveUninitialized: true
```

Comment:
can you elaborate on the secret, resave and saveUninitialized, or recommend an article/link that explains it?